docker/test/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# FIXME: once image is published inherit from official image
+# FROM cantino/huginn
+FROM singleprocess_huginn_web
```

Comment:
Does it have to derive from `cantino/huginn` or could it also be the smaller `single-process` image?